test/integration/models/test_linode.py

```diff
@@ -311,18 +316,38 @@ def test_linode_resize_with_migration_type(
     wait_for_condition(10, 100, get_status, linode, "running")
 
     time.sleep(5)
+
+    assert "g6-nanode-1" in str(linode.type)
+    assert linode.specs.disk == 25600
+
     res = linode.resize(new_type="g6-standard-1", migration_type=m_type)
 
-    assert res
+    if res:
+        wait_for_condition(
+            10,
+            60,
+            get_type,
+            test_linode_client.load(Instance, linode.id),
+            "g6-standard-1",
+        )
 
-    wait_for_condition(10, 300, get_status, linode, "resizing")
+        time.sleep(100)
```

Comment:
Just curious, what's the purpose of this sleep statement?

Reply:
So there is no "resizing" status anymore hence I manually ran the test steps couple times and found out that it takes about ~1 min to complete resizing

Reply:
Gotcha, thanks for clarifying! Would it make sense to [poll](https://github.com/linode/linode_api4-python/blob/dev/linode_api4/polling.py) for the `linode_resize` event instead?